tests/integ/sagemaker/jumpstart/estimator/test_jumpstart_estimator.py

```diff
@@ -234,5 +234,3 @@ def test_instatiating_estimator(mock_warning_logger, setup):
     elapsed_time = time.perf_counter() - start_time
 
     assert elapsed_time <= MAX_INIT_TIME_SECONDS
-
-    mock_warning_logger.assert_called_once()
```

Comment:
why is this removed? i think the intention is we want a single warning log cause we're using a wildcard version